client/landing/site.landing/coffee/core/utils.coffee

```diff
@@ -425,4 +418,18 @@ module.exports = utils = {
 
     input.setWidth width
 
+
+  loadRecaptchaScript: (callback) ->
+
+    return callback()  if grecaptcha?
+
+    recaptchaLoadCallbacks.push callback
+    recaptchaScript = new kd.CustomHTMLView
+      tagName    : 'script'
+      attributes :
+        src      : RECAPTCHA_JS
+        async    : yes
+        defer    : yes
+
+    recaptchaScript.appendToDomBody()
```

Comment:
you're right, @gokmen, I didn't take into account that script can be requested during the time it's being loaded but is not loaded yet